docs/excel/performance.md

```diff
@@ -155,11 +155,18 @@ Excel.run(async (ctx) => {
 ```
 
 > [!NOTE]
-> You can conveniently convert a Table object to a Range object by using the [Table.convertToRange()](https://dev.office.com/reference/add-ins/excel/table#converttorange) method.
+> You can conveniently convert a Table object to a Range object by using the [Table.convertToRange()](https://docs.microsoft.com/javascript/api/excel/excel.table#converttorange--) method.
+
+## Enable and disable events
+
+> [!NOTE]
+> This feature is currently available only in public preview (beta). To use it, you must reference the beta library of the Office.js CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
+
+You can disable events to improve performance. A code sample demonstrating this toggle is in the [Work with Events](excel-add-ins-events.md#enable-and-disable-events) article.
```

Comment:
Do we know for sure that disabling events ***always*** improves performance?  If not, we might want to phrase this more along the lines of "In some scenarios, performance of an add-in may be improved by disabling events."

Also, suggest replacing "demonstrating this toggle" with "that shows how to enable and disable events".